ts/Core/Chart/ChartDefaults.ts

```diff
@@ -992,13 +994,65 @@ const ChartDefaults: ChartOptions = {
      * However, `zoomBySingleTouch` will interfere with touch-dragging the
      * chart to read the tooltip. And especially when vertical zooming is
      * enabled, it will make it hard to scroll vertically on the page.
-     * @since 9.0.0
+     * @since      9.0.0
      * @sample     highcharts/chart/zoombysingletouch
      *             Zoom by single touch enabled, with buttons to toggle
      * @product    highcharts highstock gantt
+     * @deprecated
      */
     zoomBySingleTouch: false,
 
+    /**
+     * Chart zooming options.
+     * @since       next
+     */
+    zooming: {
+        /**
+         * Equivalent to [type](#chart.zooming.type), but for multitouch
+         * gestures only. By default, the `pinchType` is the same as the
+         * `type` setting. However, pinching can be enabled separately in
+         * some cases, for example in stock charts where a mouse drag pans the
+         * chart, while pinching is enabled. When [tooltip.followTouchMove](
+         * #tooltip.followTouchMove) is true, pinchType only applies to
+         * two-finger touches.
+         *
+         * @type       {string}
+         * @default    {highcharts} undefined
+         * @default    {highstock} x
+         * @product    highcharts highstock gantt
+         * @since      next
+         * @validvalue ["x", "y", "xy"]
+         * @apioption  chart.zooming.pinchType
+         */
+
+        /**
+         * Enables zooming by a single touch, in combination with
+         * [chart.zooming.type](#chart.zooming.type). When enabled, two-finger
+         * pinch will still work as set up by [chart.zooming.pinchType]
+         * (#chart.zooming.pinchType). However, `singleTouch` will interfere
+         * with touch-dragging the chart to read the tooltip. And especially
+         * when vertical zooming is enabled, it will make it hard to scroll
+         * vertically on the page.
+         * @sample     highcharts/chart/zoombysingletouch
+         *             Zoom by single touch enabled, with buttons to toggle
+         * @product    highcharts highstock gantt
+         * @since      next
+         */
+        singleTouch: false,
+
+        /**
+         * Decides in what dimensions the user can zoom by dragging the mouse.
+         * Can be one of `x`, `y` or `xy`.
+         *
+         * @type       {string}
+         * @product    highcharts highstock gantt
+         * @validvalue ["x", "y", "xy"]
+         * @since      next
+         * @apioption  chart.zooming.type
+         */
+        type: 'x'
```

Comment:
Is this new default value correct? [The current default is undefined](https://api.highcharts.com/highcharts/chart.zoomKey).

Reply:
Mistake, type should default to undefined.